lib/fcm.rb

```diff
@@ -81,8 +81,8 @@ def remove_registration_ids(key_name, project_id, notification_key, registration
       'project_id' => project_id
     }
 
-    for_uri(GROUP_NOTIFICATION_BASE_URI, extra_headers) do |connection|
-      response = connection.post('/gcm/notification', post_body.to_json)
+    for_uri(BASE_URI, extra_headers) do |connection|
```

Comment:
since there are four places where we need to call ```/fcm/notification```

- create_notification_key
- add_registration_ids
- remove_registration_ids
- recover_notification_key

maybe it would be better if we refactor this api call into a separate private method, like what ```execute_notification``` method does..

```ruby
def update_group_messaging_setting
  for_uri(BASE_URI, extra_headers) do |connection|
      response = connection.post('/fcm/notification', post_body.to_json)
      build_response(response)
  end
end
```